sql-statements/sql-statement-create-sequence.md

```diff
@@ -97,12 +98,12 @@ CREATE [TEMPORARY] SEQUENCE [IF NOT EXISTS] sequence_name
     Query OK, 0 rows affected (0.06 sec)
     ```
 
-+ 使用 `nextval()` 函数获取序列对象的下一个值。
++ 使用 `NEXTVAL()` 函数获取序列对象的下一个值。
 
     {{< copyable "sql" >}}
 
     ```sql
-    SELECT nextval(seq);
+    SELECT NEXTVAL(seq);
     ```
 
     ```
```

Comment:
L111:
```
    | NEXTVAL(seq) |
```

Reply:
updated in https://github.com/pingcap/docs-cn/pull/17220/commits/12d2ef48b19c0367986a8dfb477a72831a6a5dcb